torch/_torch_docs.py

```diff
@@ -1638,6 +1638,66 @@ def merge_dicts(*dicts):
     False
 """)
 
+add_docstr(torch.cov, r"""
+cov(input, *, correction=1, fweights=None, aweights=None) -> Tensor
+
+Estimates a covariance matrix, given :attr:`input` and weights (optional). The unbiased sample covariance
+of the variables :math:`x` and :math:`y` is given by
+
+.. math::
+    \text{cov}_w(x,y) = \frac{\sum^{N}_{i = 1}(x_{i} - \bar{x})(y_{i} - \bar{y})}{N~-~1}
+
+where :math:`x` and :math:`y` are rows of :attr:`input`, :math:`\bar{x}` and :math:`\bar{y}` are
+the simple means of the variables respectively.
+If :attr:`fweights` and/or :attr:`aweights` are provided, the unbiased weighted covariance
+is calculated, which is given by
+
+.. math::
+    \text{cov}_w(x,y) = \frac{\sum^{N}_{i = 1}w_i(x_{i} - \mu_x^*)(y_{i} - \mu_y^*)}{\sum^{N}_{i = 1}w_i~-~1}
+
+where, :math:`w` denotes :attr:`fweights` or :attr:`aweights` based on whichever is provided as an input (:math:`w = fweights \times aweights`
+if both are provided) and :math:`\mu_x^* = = \frac{\sum^{N}_{i = 1}w_ix_{i} }{\sum^{N}_{i = 1}w_i}`
+is the weighted mean of the variable.
+
+
+Args:
+    input (Tensor): A 1-D or 2-D tensor containing multiple variables and observations.
+        Each row of :attr:`input` represents a variable, and each column, a
+        single observation of all those variables. Also see :attr:`rowvar` below.
```

Comment:
rowvar is gone

the discussion of how the shape relates to the mathematical perspective might need to be moved to the start of the documentation instead of appearing here